priv/couchdb/schemas/screenpops.json

```
@@ -59,6 +59,33 @@
             },
             "type": "array"
         },
+        "permissions": {
+            "default": {},
+            "properties": {
+                "all_users": {
+                    "default": true,
+                    "description": "Whether the screenpop allowed for all users in the account",
+                    "type": "boolean"
+                },
+                "allow": {
+                    "default": [],
+                    "description": "Specific list of users who are allowed the screenpop",
+                    "items": {
```

Per our discussion, please set maximum array size (50):

```suggestion
                    "maxItems": 50,
                    "items": {
```

---

This is done